.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -1,7 +1,13 @@
-## Background
-<!-- What would a code reviewer, or a future dev, need to know about this PR in order to understand why this PR is necessary -->
+## Description of change
+Please include a description of the change. What would a code reviewer, or a future dev, need to know about this PR in order to understand why this PR is necessary.
```

Comment:
can we leave the html comments in? that way the user doesn't need to delete it every time

Reply:
Will change to be html comments.